opus/application/apps/ui/views.py

```diff
@@ -55,7 +55,8 @@
                              HTTP404_BAD_OR_MISSING_REQNO,
                              HTTP404_NO_REQUEST)
 from tools.file_utils import (get_pds_preview_images,
-                              get_pds_products)
+                              get_pds_products,
+                              get_displayed_browse_products)
```

Comment:
get_displayed_browse_products should be first since imports are kept in alphabetical order.